### ISSUE TITLE
Add BX help rendering

### ISSUE THREAD

**Issue by mdeknowis** [IBM-Cloud/ibm-cloud-cli-sdk#136]
Hello,

we try to create a BX plugin.

In your documentation is mentioned, that a command is incorrect usage should print out the help page:
https://github.com/IBM-Cloud/ibm-cloud-cli-sdk/blob/master/docs/plugin_developer_guide.md#25-incorrect-usage

The help page is defined by that structure of yours:
https://github.com/IBM-Cloud/ibm-cloud-cli-sdk/blob/master/docs/plugin_developer_guide.md#11-register-a-new-plug-in
--- SNIP ---
func (p *FssPlugin) GetMetadata() plugin.PluginMetadata {
...
--- SNAP ---

But there is no method to render manually (as mentioned in the dev guide), as it's not a good pattern to maintain data twice, and bx is able to render the help out, we would like to use a function here. Until then, we will ignore that best practice.

Regards,
Michael Decker

**Comment by izanmubarak** [comment]
@mdeknowis Were you able to figure out a solution to this? Attempting to implement this myself right now.

**Comment by mdeknowis** [comment]
@izanmubarak No. We don't render any help, we just print out, that the user has to use the help command. We don't want to reimplement, the help render engine, which is implemented by IBM. As long IBM does not provide any support method, we will not render any help output.
